### ISSUE TITLE
Benchmarking with pytorch-tabnet ?

### ISSUE THREAD

**Issue by Optimox** [Kaixhin/SARCOS#1]
Hello,

I was wondering if you'd like to add pytorch-tabnet to your comparison? 
https://github.com/dreamquark-ai/tabnet

Let me know if this is of interest and if you need help.

Thanks!

**Comment by Kaixhin** [comment]
As discussed [here](https://www.datarobot.com/blog/running-code-and-failing-models/), the SARCOS dataset has data leakage and shouldn't be used in its current form; this has also been noted prominently in the README. I am keeping this repository as a reference, but because of this issue won't be changing anything.

**Comment by Optimox** [comment]
Ok no problem.

I was wondering why XGBoost could not overfit the test set, a very deep XGBoost should overfit just perfectly the test set.

So I don't understand how you chose the parameters for each model and how the current results might be considered a reference in any way (even considering the leakage between train and test set).

**Comment by Kaixhin** [comment]
Reference for [Adaptive Neural Trees](http://proceedings.mlr.press/v97/tanno19a.html). I'm not particularly familiar with the details of XGBoost, and stuck to the default implementations in scikit-learn. scikit-learn's tree-based models do not scale so well, so I was also limited by the hardware I had available at the time (a personal laptop).
